target/linux/ath79/image/generic.mk

```diff
@@ -798,6 +798,17 @@ define Device/comfast_cf-e314n-v2
 endef
 TARGET_DEVICES += comfast_cf-e314n-v2
 
+define Device/comfast_cf-e355ac-v2
+  SOC := qca9531
+  DEVICE_VENDOR := COMFAST
+  DEVICE_MODEL := CF-E355AC
+  DEVICE_VARIANT := v2
+  DEVICE_PACKAGES := kmod-ath10k-ct ath10k-firmware-qca9888-ct \
+	-swconfig -uboot-envtools
```

Comment:
why uboot-envtools?

Reply:
I was mostly following how other Comfast devices have been set up, though as this device uses u-boot, it might make sense to not drop it..

Reply:
If you want to keep uboot-envtools, define the partition in dts. It’ll work when @rmilecki fixes uboot-envtools.

Reply:
If it is not necessary for the basic usage of the device, it is safe to remove it from default packages.

Reply:
Yeah, I would rather keep it as is for now, because even after bricking and recovering this device multiple times, I haven't had the need to make changes to u-boot configuration.